app/services/appointment_notification_service.rb

```diff
@@ -12,9 +12,10 @@ def initialize(appointments:, days_overdue: 3, schedule_at:)
 
   def send_after_missed_visit
     overdue_with_patient_consent = appointments.overdue_by(days_overdue)
```

Comment:
We could extract out `overdue_with_patient_consent` in its own scope on `Appointment`.

Reply:
Would `eligible_for_reminder` be a better name?

Reply:
Yup.